libs/entitlement/qa/test-plugin/build.gradle

```diff
@@ -7,16 +7,22 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import org.elasticsearch.gradle.internal.precommit.CheckForbiddenApisTask
-
+apply plugin: 'elasticsearch.base-internal-es-plugin'
 apply plugin: 'elasticsearch.build'
 apply plugin: 'elasticsearch.mrjar'
 
+esplugin {
+  name = 'test-plugin'
+  description = 'A test plugin that invokes methods checked by entitlements'
+  classname = 'org.elasticsearch.entitlement.qa.test.EntitlementTestPlugin'
+}
+
 dependencies {
   implementation project(':server')
   implementation project(':libs:logging')
 }
 
-tasks.withType(CheckForbiddenApisTask).configureEach {
-  replaceSignatureFiles 'jdk-signatures'
+tasks.named("javadoc").configure {
```

Comment:
I think the CheckForbiddenApisTask fails with the mrjar plugin under some circumstances, but if that's not true happy to remove it.